addons/website/static/src/interactions/dropdown/dropdown_mega_menu.js

```diff
@@ -120,4 +120,4 @@ class MegaMenuDropdown extends Interaction {
 
 registry
     .category("website.active_elements")
-    .add("website.mega_menu_dropdown", MegaMenuDropdown);
+    .add("website.dropdown_mega_menu", DropdownMegaMenu);
```

Comment:
Wondering... :thinking:  wasn't the old name better ?

Reply:
True, I switch it back